composer.json

```diff
@@ -16,8 +16,8 @@
 		"lcobucci/jwt": "~4.1 || ~5.0"
 	},
 	"require-dev": {
-		"phpunit/phpunit": ">=9.0 <9.6",
-		"lunr/halo": "~0.8.0"
+		"phpunit/phpunit": "~10.0",
+		"lunr/halo": "~0.10.0"
```

Comment:
Please do the update to Lunr.Halo 0.10 in a separate PR. It has nothing to do with live activities